src/stylesheets/components/_footer.scss

```diff
@@ -299,8 +326,15 @@
       justify-content: flex-start;
 
       // Arrow for collapsible content.
+      @include place-icon(
+        $chevron-expand-more,
+        "before",
+        0.5,
+        "auto",
+        "default"
+      );
```

Comment:
# Footer

Currently, social media icons and expand chevrons (mobile only) disappear in dark high contrast modes, causing the user to lose context and content.

This PR also fixes the disappearing chevrons in the big footer variant in mobile views.


## Social Icons

Currently: 
(Alternative color shows hover)

_Default_

![image](https://user-images.githubusercontent.com/25211650/154766248-ebaafd5a-6544-47ba-9a0d-dd8322ea4f43.png)


_High Contrast_

![image](https://user-images.githubusercontent.com/25211650/154765695-351ae222-a80b-4b03-a87e-2c77993324e0.png)


After Fix:

Used `forced-color-adjust: none` to set the background color for the social link icons to lightgrey. This is easy to see in all high contrast modes and keeps things consistent between versions

![image](https://user-images.githubusercontent.com/25211650/160169166-2824eebe-08de-4f14-8acd-47105b02217b.png)


### Mobile Chevrons

Currently:

![image](https://user-images.githubusercontent.com/25211650/155201505-6c769bf8-3814-4a3a-ba3d-3cbe657e7739.png)

After Fix:

![image](https://user-images.githubusercontent.com/25211650/155202083-c9e62253-7fdd-4ff5-aa50-be53457b6d70.png)